astropy/timeseries/periodograms/lombscargle/core.py

```diff
@@ -399,15 +399,15 @@ def model(self, t, frequency):
 
         Parameters
         ----------
-        t : array_like or Quantity, length n_samples
-            times at which to compute the model
+        t : array_like or `~astropy.units.Quantity`.
+            Length ``n_samples`` times at which to compute the model.
```

Comment:
Maybe to be more readable:

Times at which to compute the model (should have length ``n_samples``)

Reply:
✅